python_scripts/shellies_discovery_gen2.py

```diff
@@ -4796,6 +4806,58 @@ def get_binary_sensor(
     return topic, payload
 
 
+def get_bthome_event(bt_id, device_id_prefix):
+    """Create configuration for BTHome device event entity."""
+    topic = encode_config_topic(
+        f"{disc_prefix}/event/{device_id_prefix}-bthome-{bt_id}/config"
+    )
+
+    # For BTHome devices, we'll use a generic button name since they don't have
+    # individual input names like regular Shelly devices
+    button_name = f"BTHome Button {bt_id}"
+
+    payload = {
+        KEY_NAME: button_name,
```

Comment:
We don't need BTHome prefix here
```suggestion
    payload = {
        KEY_NAME:  f"Button {bt_id}",
```